dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -242,6 +242,19 @@ export const ImageComponent = ({
 
 	const palette = decidePalette(format);
 
+	const master = element.media.allImages.find(
+		(image) => image.fields.isMaster,
+	)?.url;
+
+	if (!master) {
```

Comment:
I think `main` is becoming more the norm now instead of `master`? Or `source`?

Reply:
I think consistency with how the Grid refers to this concept is important.  Unilaterally changing to `main` or `source` here would cause confusion.

Reply:
Thanks for raising this. I agree with @davidfurey we probably don't want to diverge just in DCR but I also think it would be good to try and change this so I raised the question over in The Grid channel. Maybe there's already been some thinking around this.